salt_stack/salt/boss/django.sls

```diff
@@ -28,10 +28,26 @@ django-files:
         - source: salt://boss/files/boss.git
         - include_empty: true
 
+ssl-config:
+    file.managed:
+        - mode: 600
+        - names:
+            - /etc/ssl/certs/nginx-selfsigned.crt:
+                - source: salt://boss/files/boss.git/nginx-selfsigned.crt
```

Comment:
Maybe makes more sense to store these in the `files` parent folder since they aren't really part of the `boss` repo?

Reply:
Good idea, just pushed!